src/sentry/utils/sdk_crashes/event_stripper.py

```diff
@@ -34,6 +34,10 @@ def strip_event_data(self, event: Event) -> Event:
         stripped_frames = self._strip_frames(frames)
         new_event["exception"]["values"][0]["stacktrace"]["frames"] = stripped_frames
 
+        debug_meta_images = get_path(event, "debug_meta", "images")
+        stripped_debug_meta_images = self._strip_debugMeta(debug_meta_images, stripped_frames)
+        new_event["debug_meta"]["images"] = stripped_debug_meta_images
```

Comment:
AFAIK `get_path` might return `None`:

```suggestion
    if debug_meta_images:
            stripped_debug_meta_images = self._strip_debugMeta(debug_meta_images, stripped_frames)
            new_event["debug_meta"]["images"] = stripped_debug_meta_images
```